guides/common/modules/proc_configuring-the-discovery-service.adoc

```diff
@@ -37,27 +37,37 @@ endif::[]
 hosts or virtual machines can be created with incorrect network configurations. On Dell servers, the new naming scheme can be disabled via `biosdevname=1` kernel command line option, for other hardware or virtual machines, the new naming scheme can be completely turned off via `net.ifnames=1`.
 endif::[]
 
-To use {ProjectServer} to provide the Discovery image, install the following RPM packages:
-
-* `tfm-rubygem-foreman_discovery`
-ifdef::satellite[]
-* `foreman-discovery-image`
+.Procedure
+. To use {ProjectServer} to provide the Discovery image, run the following commands:
++
+ifdef::foreman-el,katello[]
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {foreman-installer} \
+--enable-foreman-plugin-discovery \
+--enable-foreman-proxy-plugin-discovery \
+--foreman-proxy-plugin-discovery-install-images=true \
+--foreman-proxy-plugin-discovery-source-url=http://downloads.theforeman.org/discovery/releases/3.5/
+----
 endif::[]
-* `tfm-rubygem-smart_proxy_discovery`
-
-The `tfm-rubygem-foreman_discovery` package contains the {Project} plug-in to handle discovered nodes, connections, and necessary database structures, and API.
-
 ifdef::satellite[]
-The `foreman-discovery-image` package installs the Discovery ISO to the `/usr/share/foreman-discovery-image/` directory.
-You can build a PXE boot image from this ISO using the `livecd-iso-to-pxeboot` tool.
-The tool saves this PXE boot image in the `/var/lib/tftpboot/boot` directory.
-For more information, see xref:Building_a_Discovery_Image_{context}[].
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {foreman-installer} \
+--enable-foreman-plugin-discovery \
+--enable-foreman-proxy-plugin-discovery \
+--foreman-proxy-plugin-discovery-install-images=false
+----
 endif::[]
-
-The `tfm-rubygem-smart_proxy_discovery` package configures {SmartProxyServer}, such as the integrated {SmartProxy} of {ProjectServer}, to act as a proxy for the Discovery service.
+. Install `foreman-discovery-image`:
```

Comment:
We only have `foreman-discovery-image` in satellite, there is no upstream equivalent. I did see previously there it was also used for `orcharhino`. @maximiliankolb is this also something that Orcharhino duplicated?

Please move this into the `ifdef::satellite` block.

Reply:
I have considered `Orcharhino` from `guides/common/modules/proc_installing-the-discovery-service.adoc`

Correct me if I am wrong. Addressed :+1: